### ISSUE TITLE
Collecting resources for webhooks.org re-launch

### ISSUE THREAD

**Issue by johnsheehan** [progrium/webhooks#1]
As part of the [Webhooks.org relaunch I'm working on](https://groups.google.com/forum/#!topic/webhooks/yYxws_go3P4) I'd like to include a comprehensive list of links to articles, blog posts, videos, presentations, etc. about webhooks.

Please post a comment with anything you'd like included.

**Comment by johnsheehan** [comment]
7 Reasons Webhooks Are Magic
http://blog.iron.io/2013/09/7-reasons-webhooks-are-magic.html

**Comment by johnsheehan** [comment]
Using APIs to Connect People to Your App and Each Other
http://www.infoq.com/presentations/Using-APIs-to-Connect-People-to-Your-App

**Comment by julien51** [comment]
I think a lot of PubSubHubbub content would be meaningful now that it's not RSS/Atom specific.

**Comment by johnsheehan** [comment]
Agreed. Please post specific links you'd like to see included here.

**Comment by julien51** [comment]
Hey John, here is a blog post I just wrote about that: http://blog.superfeedr.com/webhooks-improved/ 
Let me know if that fits what you're looking for! Also, let me know if there is more we can do.

**Comment by defel** [comment]
resthooks http://resthooks.org/

**Comment by jkarneges** [comment]
http://webhookinbox.com/
Like requestbin but with live UI and expanded API. Can be used by browser apps to receive PubSubHubbub events.

**Comment by bpedro** [comment]
[Webhooks, REST and the Open Web](http://apiux.com/2013/09/12/webhooks/)

From Webhooks into a publish/subscribe model based on PubSubHubbub.

**Comment by johnsheehan** [comment]
Both by @davefp

http://www.shopify.com/technology/5154662-webhook-best-practices#axzz2pwD4aZ4T
http://www.shopify.com/technology/3931292-webhook-testing-made-easy

**Comment by davefp** [comment]
GitHub's webhook testing guide: https://help.github.com/articles/testing-webhooks

**Comment by johnsheehan** [comment]
http://culttt.com/2014/01/22/webhooks/

**Comment by johnsheehan** [comment]
http://blog.fanout.io/2014/01/27/how-to-safely-invoke-webhooks/

**Comment by davefp** [comment]
http://www.pittrap.com/testing-stripe-webhooks/

**Comment by BigBlueHat** [comment]
Couple posts on setting up Webhooks to "dump" into [Cloudant](http://cloudant.com/) and [Apache CouchDB](http://couchdb.apache.org/):
https://cloudant.com/blog/weblatch-connect-your-webhooks/
https://cloudant.com/blog/integrating-github-and-cloudant-using-webhooks/

Thanks for building this list!

**Comment by davefp** [comment]
New GitHub webhook guide: http://developer.github.com/webhooks/

Blog post with details of new features: https://github.com/blog/1778-webhooks-level-up

**Comment by ssaunier** [comment]
How we use GitHub "push webhook" in a code school:
http://sebastien.saunier.me/blog/2014/04/21/practical-example-of-using-git-in-a-school.html

**Comment by mapping-edges** [comment]
Muut now supports wehhooks. Just check out our blog entry and the documentation. 
https://muut.com/blog/release/v1.12.html
https://muut.com/docs/webhooks.html
Feel free to contact us for any details!

**Comment by davefp** [comment]
Sendgrid article on webhooks vs. APIs http://sendgrid.com/blog/webhook-vs-api-whats-difference/

**Comment by johnsheehan** [comment]
http://sendgrid.com/blog/whats-webhook/

We've got a lot now. I should probably get back to this project...

**Comment by johnsheehan** [comment]
http://blog.fanout.io/2015/04/02/realtime-api-design-guide/
